README.md

```diff
@@ -84,11 +117,11 @@ Step 3: Add a `dependencyManagement` block that contains the `arquillian-bom`:
 ```
 dependencyManagement {
     imports {
-        mavenBom "org.jboss.arquillian:arquillian-bom:1.1.13.Final"
+        mavenBom "org.jboss.arquillian:arquillian-bom:1.7.0.Alpha9"
     }
 }
 ```
 
 You can use the command `gradle dependencies` to view transitive dependencies and their versions. 
 
-The previous example uses version `1.1.13.Final` of the `arquillian-bom`. A full Gradle example is available in the `gradle-tests` folder. 
+The previous example uses version `1.7.0.Alpha9` of the `arquillian-bom`. A full Gradle example with Jakarta EE 9 is available in the `gradle-tests` folder.
```

Comment:
Should they use a different version for Java EE 8 as implied earlier in the readme?